open_blender_fast64.bat

```diff
@@ -0,0 +1 @@
+"c:\Program Files\Blender Foundation\Blender 4.0\blender.exe" --python tools/blender_extension/fast64_run.py
```

Comment:
This is something that's going to break when Blender updates.

Can we have remove these three shortcut files and instead have it in the makefile like for the emulator?

`make blender` or `make open_blender` or something?

Reply:
And let me know your thoughts on this idea just in-case this idea isn't good.

Reply:
to be more precise this file are not planned to stay in the final version it's just to test it can be open_blender_fast64 it's fine but fast64 can work between 3.2-4.0 so just precise the version will break with other version avaible so I will make under another way

Reply:
It's fine if you hit the latest or most versions. Doesn't need to be perfect.